src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_logicapp_commands.py

```diff
@@ -116,6 +117,139 @@ def test_logicapp_https_only(self, resource_group, storage_account):
         self.cmd(f'logicapp create -g {resource_group} -n {logicapp_name} -p {plan} -s {storage_account} --https-only', checks=[JMESPathCheck('httpsOnly', True)])
         self.cmd('logicapp create -g {} -n {} -p {} -s {}'.format(resource_group, logicapp_name, plan, storage_account), checks=[JMESPathCheck('httpsOnly', False)])
 
+    @ResourceGroupPreparer(location=DEFAULT_LOCATION)
+    def test_logicapp_config_appsettings_e2e(self, resource_group):
```

Comment:
is there a reason we need re-deifine all this logic separately? Can we just not call the functionapp methods? Ideally, we should re-use as much logic as possible since functionapps & LA are the identical under the hood.